src/containers/Questions/questions.jsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+
+import propTypes from 'prop-types';
+
+import { Button, message, Input, Progress, Icon, Popconfirm } from 'antd';
+
+import BackButton from '../../components/BackButton';
+import schema from './questionValidation';
+import './questions.css';
+
+const { TextArea } = Input;
+const journals = [];
+
+const entryData = [
+  {
+    id: 1,
+    heading: 'Today, Iam grateful for',
+    percent: 34,
+  },
+  {
+    id: 2,
+    heading: 'Challenge I face',
+    percent: 68,
+  },
+  {
+    id: 3,
+    heading: 'Iam looking to develop',
+    percent: 100,
+  },
+];
+
+class Questions extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      current: 0,
+      title: '',
+      content: '',
+      errors: {},
+    };
+  }
+
+  confirm = e => {
+    message.success("You didn't make an entry today");
+    const { history } = this.props;
+    history.push('/home');
+  };
+
+  handleChange = ({ target: { value, name } }) => {
+    this.setState({ [name]: value, errors: {} });
+  };
+
+  next = async () => {
+    const { current, title, content } = this.state;
+    try {
+      await schema.validate({ title, content }, { abortEarly: false });
+      journals.push({ title, content });
+      return this.setState({
+        current: current + 1,
+        content: '',
+        title: '',
+        errors: {},
+      });
+    } catch (error) {
+      const objError = {};
+      error.inner.forEach(fielderror => {
+        objError[fielderror.path] = fielderror.message;
+      });
+      return this.setState({ errors: objError });
+    }
+  };
+
+  finish = async () => {
+    const { title, content } = this.state;
+    const { history } = this.props;
+    try {
+      await schema.validate({ title, content }, { abortEarly: false });
+      message.success('Yes, you have added a journal');
+      return history.push('/home');
```

Comment:
I think you should also save the third journal 
```suggestion
      journals.push({title, content});
      return history.push('/home');
```